readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -0,0 +1,79 @@
+# Read the Docs configuration file
+
+# The version of the spec to be use
+version: enum('2')
+
+# Formats of the documentation to be built
+# Default: ['htmlzip', 'pdf', 'epub', 'singlehtmllocalmedia']
+formats: any(list(enum('htmlzip', 'pdf', 'epub', 'singlehtmllocalmedia')), enum('all'), required=False)
+
+# The path to the requirements file from the root of the project
+requirements_file: str(required=False)
+
+# Configuration for Conda support
+conda: include('conda', required=False)
+
+# Configuration for the documentation build process
+build: include('build', required=False)
+
+# Configuration of the Python executable to be used
+python: include('python', required=False)
+
+# Configuration for sphinx
+sphinx: include('sphinx', required=False)
+
+# Submodules configuration
+submodules: include('submodules', required=False)
+
+# Redirects for the current version to be built
+redirects: include('redirects', required=False)
+
+---
+
+conda:
+  # The path to the Conda environment file from the root of the project
+  file: str()
```

Comment:
is it possible to add some kind of validation here? I mean, it's not just any string, but a relative PATH

Reply:
maybe it's more expressive to use `environment` instead of `file`, or `environment_file` similar to `requirements_file`.

Or, instead of `_file` we can use `_path`.

I prefer without the `_file`, though. Just `requirements` and `environment`.

Reply:
> I prefer without the _file, though. Just requirements and environment.

Yeah, I agree with that.

For the validation, there isn't a path type or something like that, we can extend it and implement one though. I'll try to see what I can do here.